libraries/AP_Common/AP_Common.cpp

```diff
@@ -101,3 +101,39 @@ int16_t char_to_hex(char a)
     else
         return a - '0';
 }
+
+int32_t to_int32(const float value)
```

Comment:
maybe float_to_int32_le() ? and make it a const float reference, so avoid auto-promotion

Reply:
Done in dce44d9839f3cd53e1596d8adcbfa4d91cc4191d